docker-compose.yaml

```diff
@@ -0,0 +1,225 @@
+version: '3'
+
+networks:
+  internal:
+    external: false
+
+volumes:
+  objectstoredata: {}
+  postgres: {}
+  waitforit: {}
+
+services:
+  # #
```

Comment:
It is "in utero" code instead of "dead" code :smile:, where I was beginning to explore how to also run scimma-admin locally to make the entire system self-contained and not reliant on external services. We could shove this into a branch if it looks too ugly, but it doesn't cause any harm as is. Maybe I could add a note to this effect as a comment at the top of the block, and then move the block to the bottom?

Reply:
I prefer storing in-progress code in a branch if it isn't ready yet.